shared/router-v2/shim.native.tsx

```diff
@@ -46,33 +40,10 @@ const platformShim = (
         </Kb.KeyboardAvoidingView2>
       )
     }
-
-    if (isModal) {
-      wrap = <ModalWrapper navigationOptions={navigationOptions}>{wrap}</ModalWrapper>
-    }
     return wrap
   })
 }
 
-const ModalWrapper = (p: {children: React.ReactNode; navigationOptions: GetOptionsRet}) => {
```

Comment:
I think this doesn't make sense, its not aware of the difference between the keyboard being up or not so the spacing won't work.